content/en/docs/concepts/storage/storage-limits.md

```diff
@@ -32,6 +32,7 @@ that can be attached to a Node:
   <tr><td><a href="https://aws.amazon.com/ebs/">Amazon Elastic Block Store (EBS)</a></td><td>39</td></tr>
   <tr><td><a href="https://cloud.google.com/persistent-disk/">Google Persistent Disk</a></td><td>16</td></tr>
   <tr><td><a href="https://azure.microsoft.com/en-us/services/storage/main-disks/">Microsoft Azure Disk Storage</a></td><td>16</td></tr>
+  <tr><td><a href="https://www.digitalocean.com/docs/kubernetes/how-to/add-volumes/">Digital Ocean Block Storage Volume</a></td><td>7</td></tr>
```

Comment:
Hi, unfortunately we only document in-tree volume types on this page. For CSI volume types, we could add a note saying something like "Check with your CSI driver documentation for specific limits" and point to the csi driver page https://kubernetes-csi.github.io/docs/drivers.html